app/templates/admin/sales/discounted-events.hbs

```diff
@@ -1 +1,113 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h1 class="ui header six wide column">{{t 'Tickets Summary'}}</h1>
```

Comment:
Please use `h2` here similar to `events/<id>/tickets`. Also we do not really require `six wide` here as we only have the heading here, It will cause the text to wrap due to limited space for tablet devices.